src/main/java/org/fundacionjala/coding/franz/movies/Customer.java

```diff
@@ -0,0 +1,73 @@
+package org.fundacionjala.coding.franz.movies;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * class of custumer.
+ */
+public class Customer {
+    private String name;
+    private List<Rental> listRental;
+
+
+    /**
+     * This is the constructor.
+     *
+     * @param name customer.
+     */
+    public Customer(final String name) {
+        this.name = name;
+        listRental = new ArrayList<Rental>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)